pages/initiatives/gsoc/gsoc2021ideas.md

```diff
@@ -96,3 +96,33 @@ list? Great, please submit it!
 
 ----
 
+
+### [OWASP Maryam](https://github.com/saeeddhqan/maryam)
+
+##### Explanation of Ideas
+OWASP Maryam is a modular open-source OSINT based framework. Maryam is written in Python and It’s designed to provide a powerful environment to harvest data from open-sources and search-engines and collect data quickly and thoroughly.
+
+##### Getting started
+
+* You may read the available documents in the [wiki page](https://github.com/saeeddhqan/maryam/wiki). Especially, Development Guide section.
+* Any question, problem, and discussion? contact with [Saeed Dehqan](mailto:saeed.dehghan@owasp.org).
```

Comment:
Is your email address actually a different spelling vs your name? (Below too)

I understand it totally could be, it just seemed out of place and caught my eye.